src/apps/alert/AlertReceiver.cc

```diff
@@ -45,17 +46,23 @@ void AlertReceiver::handleMessage(cMessage *msg)
 {
     if (msg->isSelfMessage())
         return;
-    AlertPacket* pkt = check_and_cast<AlertPacket*>(msg);
 
-    if (pkt == 0)
-        throw cRuntimeError("AlertReceiver::handleMessage - FATAL! Error when casting to AlertPacket");
+    Packet* pPacket = check_and_cast<Packet*>(msg);
+
+    if (pPacket == 0)
```

Comment:
check_and_cast cannot return a nullptr, so there's no point in checking the result

Reply:
Has been removed